### DISCUSSION TITLE
Why print & pprint command auto-fix is not available ?

### DISCUSSION THREAD

**Discussion by jiaulislam** [astral-sh/ruff#9204] [General]
As per documentation I see there's a rule for print statements `T20`. But it's not auto fixable. Can't we make it auto fixable ?

**Comment by charliermarsh** [comment]
We could add a fix for this!

**Reply by charliermarsh** [reply]
Tracking here: https://github.com/astral-sh/ruff/issues/9207. I'll implement it now.

**Reply by charliermarsh** [reply]
Added in https://github.com/astral-sh/ruff/pull/9208. Will go out in the next release. Thanks!

**Reply by jiaulislam** [reply]
wow ! that was fast. thankyou so much
